.github/workflows/format.yml

```diff
@@ -58,4 +58,4 @@ jobs:
           dprint check
 
       - name: cargo-license-template check
-        run: cargo license-template --template .license_template --ignore .license_template_ignore --verbose
+        run: cargo +nightly license-template --template .license_template --ignore .license_template_ignore --verbose
```

Comment:
Probably doesn't matter if run with `nightly` because license-template should be agnostic over the environment, but also shouldn't be a problem.